.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
also include yarn.lock in gitignore?

Reply:
The `yarn.lock` file should be in source control.

[As per the Yarn docs](https://classic.yarnpkg.com/en/docs/migrating-from-npm):
> You don’t need to read or understand this file - just check it into source control.